flocker/testtools/strategies.py

```diff
@@ -0,0 +1,49 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+import string
+
+from hypothesis.strategies import lists, text
+from twisted.python.filepath import FilePath
+
+
+path_segments = (
+    text().
+    filter(lambda x: '/' not in x).
+    map(lambda x: x.encode('utf8')).
+    filter(lambda x: '\0' not in x))
+"""
+Individual path segments.
+
+These are UTF-8 encoded segments that contain neither '/' nor NULL.
+
+e.g. 'foo', 'rc.local'.
+"""
+
+
+paths = lists(path_segments).map(lambda ps: FilePath('/'.join(ps)))
+"""
+Paths
+
+e.g. '/usr/local', 'foo/bar/bar'.
+"""
```

Comment:
`FilePath` always abspaths what it gets, so it's probably OK. I've updated the examples to include `FilePath`.